docs/yaml_docs/pipeline_clustering_yml.md

```diff
@@ -228,7 +228,7 @@ When pseudo_seurat is set to True then a [python implementation](https://github.
     - <span class="parameter">layer </span> `String`, Default: logged_counts<br>
       Which layer stores counts for differential expression test. 
        Options include logged_counts, signac_norm , and logTF_norm,logIDF_norm
-    - <span class="parameter">mincels </span> `Integer`, Default: t-10<br>
+    - <span class="parameter">mincels </span> `Integer`, Default: 10<br>
        If the number of clusters contains less than the number of cells maker analysis is not necessary.
```

Comment:
Min number of cells in a cluster. If the cluster contains less than this number of cells, the marker analysis won't be run.